features/auth/Logout.tsx

```diff
@@ -16,7 +16,7 @@ export default function Logout() {
   const logout = useMutation<void, RpcError>(
     async () => {
       authActions.logout();
-      queryClient.resetQueries();
+      queryClient.clear();
```

Comment:
It doesn't seem to be too slow for React Query to re-establish the cleared queries, so better be safe than micro-optimising by only resetting the query previously!